### PR TITLE
Design Tokens - Wizard

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
Design Tokens - Wizard
WOAUILIB-2724

## Changelog
Design Tokens - Wizard

### CODE REVIEW

File: src/components/wizard/WizardStates.ts [hunk 1]
```diff
@@ -4,16 +4,27 @@ const exclamationSmall = require('./assets/exclamationSmall.png');
 import {WizardStepsConfig} from './types';
 
 export const StatesConfig: WizardStepsConfig = {
-  enabled: {color: Colors.grey30, circleColor: Colors.grey60, enabled: true},
-  disabled: {color: Colors.grey50, circleColor: Colors.grey60},
-  error: {color: Colors.red30, icon: exclamationSmall, enabled: true, accessibilityInfo: 'Validation Error'},
-  skipped: {color: Colors.red30, enabled: true, accessibilityInfo: 'Not completed'},
+  enabled: {color: Colors.$textNeutralHeavy, circleColor: Colors.$outlineDisabled, enabled: true},
+  disabled: {color: Colors.$textDisabled, circleColor: Colors.outlineDefault},
```
Comment (Inbal-Tish) [inline src/components/wizard/WizardStates.ts:8]:
```suggestion
  disabled: {color: Colors.$textDisabled, circleColor: Colors.$outlineDisabled},
```

File: src/components/wizard/WizardStates.ts [hunk 2]
```diff
@@ -4,16 +4,27 @@ const exclamationSmall = require('./assets/exclamationSmall.png');
 import {WizardStepsConfig} from './types';
 
 export const StatesConfig: WizardStepsConfig = {
-  enabled: {color: Colors.grey30, circleColor: Colors.grey60, enabled: true},
-  disabled: {color: Colors.grey50, circleColor: Colors.grey60},
-  error: {color: Colors.red30, icon: exclamationSmall, enabled: true, accessibilityInfo: 'Validation Error'},
-  skipped: {color: Colors.red30, enabled: true, accessibilityInfo: 'Not completed'},
+  enabled: {color: Colors.$textNeutralHeavy, circleColor: Colors.$outlineDisabled, enabled: true},
```
Comment (Inbal-Tish) [inline src/components/wizard/WizardStates.ts:7]:
```suggestion
  enabled: {color: Colors.$textNeutralHeavy, circleColor: Colors.$outlineDefault, enabled: true},
```

File: src/components/wizard/WizardStates.ts [hunk 3]
```diff
@@ -4,16 +4,27 @@ const exclamationSmall = require('./assets/exclamationSmall.png');
...
+    color: Colors.$textDangerLight,
+    circleColor: Colors.$outlineDanger,
+    enabled: true,
+    accessibilityInfo: 'Not completed'
+  },
   completed: {
-    color: Colors.grey30,
-    circleColor: Colors.grey60,
+    color: Colors.$iconNeutral,
+    circleColor: Colors.$outlineDisabled,
```
Comment (Inbal-Tish) [inline src/components/wizard/WizardStates.ts:24]:
```suggestion
    circleColor: Colors.$outlineDefault,
```

Comment (Inbal-Tish) [reply src/components/wizard/WizardStates.ts:24]: Was it changed to grey50 so you're using the $outlineDisabled token?

File: src/components/wizard/WizardStates.ts (hunk 2 above)
Comment (Inbal-Tish) [reply src/components/wizard/WizardStates.ts:7]: Was it changed to grey50 so you're using the $outlineDisabled token?

File: src/components/wizard/WizardStates.ts (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/wizard/WizardStates.ts:8]: Was it changed to grey60 so you're using the $outlineDefault token? anyways your missing the '$'

File: src/components/wizard/WizardStep.tsx [hunk 4]
```diff
@@ -118,7 +118,7 @@ export default asBaseComponent<WizardStepProps>(WizardStep);
 const styles = StyleSheet.create({
   connector: {
     borderWidth: 0.5,
-    borderColor: Colors.grey60
+    borderColor: Colors.$outlineDisabled
```
Comment (Inbal-Tish) [inline src/components/wizard/WizardStep.tsx:121]: Was it changed to grey50 so you're using the $outlineDisabled token?

File: src/components/wizard/WizardStates.ts (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/wizard/WizardStates.ts:8]: There is a new design page for tokenized components: [Wizard](https://www.figma.com/file/HzaNWJziTYvLgU2vzQWjKg/DS-Tokenized?node-id=6457%3A400750) 
It's from there

File: src/components/wizard/WizardStates.ts (hunk 2 above)
Comment (M-i-k-e-l) [reply src/components/wizard/WizardStates.ts:7]: It's set to $outlineNeutralMedium which is $outlineDisabled

File: src/components/wizard/WizardStates.ts (hunk 3 above)
Comment (M-i-k-e-l) [reply src/components/wizard/WizardStates.ts:24]: It's set to $outlineNeutralMedium which is $outlineDisabled

File: src/components/wizard/WizardStep.tsx (hunk 4 above)
Comment (M-i-k-e-l) [reply src/components/wizard/WizardStep.tsx:121]: See other comments :)

File: src/components/wizard/WizardStates.ts (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/wizard/WizardStates.ts]: I see there both disabled and enabled borderline have the same token...

Comment (M-i-k-e-l) [reply src/components/wizard/WizardStates.ts]: Look at the bottom part, all the types are there.
You might have looked at `enabled` and `completed`.

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**